#goal   
geometry_msgs/Pose tool_to_obj
geometry_msgs/Point delta
std_msgs/Float64MultiArray hand_preshape
std_msgs/Float64MultiArray hand_target
float64 alpha
float64 max_vel
float64 max_angvel
float64 max_accel
float64 max_angaccel
---
#result
int64 success
---
#feedback
int64 percentage